apps/next/src/pages/Onboarding/components/CardMenu.tsx

```diff
@@ -39,22 +42,32 @@ export const CardMenuItem: FC<CardMenuItemProps> = ({
   icon,
   text,
   description,
+  size,
   ...props
 }) => {
   const history = useHistory();
+  const theme = useTheme();
 
   const onClick =
     'onClick' in props ? props.onClick : () => history.push(props.link);
 
   return (
-    <CardMenuItemContainer onClick={onClick}>
+    <CardMenuItemContainer
+      onClick={onClick}
+      sx={{
+        gap: size === 'small' ? theme.spacing(1.5) : theme.spacing(3),
+        paddingX: size === 'small' ? theme.spacing(1.5) : theme.spacing(2.5),
+      }}
```

Comment:
Please consider taking a more concise approach.

```suggestion
        gap: theme.spacing(size === 'small' ? 1.5 : 3),
        paddingX: theme.spacing(size === 'small' ?  1.5 : 2.5),
      }}
```